validationFunctions/validatePartnerName.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Function for validating the partner name mandatory field
+ * @module functions/validatePartnerName
+ */
+
+const { isEmpty, trim } = require("validator");
+
+/**
+ * Runs custom validation on the partner name. It will be true if the input is a string less than 56 characters. Empty string will return false as the field is mandatory.
+ *
+ * @param {string} name The text string of the partner name the user supplies
+ *
+ * @returns {boolean} It will return true if the string is valid and false if it is not valid
+ */
+
+const validatePartnerName = name => {
```

Comment:
is there an argument for parametering a validateStringLength function to cover these kind of checks or would it not work with how it's invoked?

Reply:
This project doesn't have any helper functions; each validation function is independent. The validator package is in use, however its isLength function has not been used before.

Reply:
We'll pick thsi up should we add more length validation in the future - not a priority now.